lib/virt_feature_test_base.pm

```diff
@@ -160,7 +160,12 @@ sub post_fail_hook {
     #(caller(0))[3] can help pass calling subroutine name into called subroutine
     $self->junit_log_provision((caller(0))[3]) if get_var("VIRT_AUTOTEST");
 
-    virt_utils::collect_host_and_guest_logs;
+    if (get_var('ENABLE_VIR_NET')) {
+        virt_utils::collect_host_and_guest_logs('', '', '/tmp/wickded');
+    }
+    else {
+        virt_utils::collect_host_and_guest_logs;
+    }
     upload_logs("/var/log/clean_up_virt_logs.log");
     save_screenshot;
     upload_logs("/var/log/guest_console_monitor.log");
```

Comment:
Is this the only file you are going to modify or there are more files to be modified ? You'd better set WIP if there is ongoing work.

Reply:
sure, had updated as WIP now